backend/main.py

```
@@ -895,6 +927,32 @@ async def login_user(user: UserLogin, response: Response, db: Session = Depends(
         )
     )
 
+@app.post("/users/forgot-password")
+async def forgot_password(request: PasswordResetRequest, db: Session = Depends(get_db)):
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)